src/newrelic/processor.go

```diff
@@ -544,14 +579,17 @@ func harvestPayload(p PayloadCreator, args *harvestArgs) {
 		}),
 	}
 
-	reply := args.client.Execute(cmd, cs)
+	reply := args.client.Execute(&cmd, cs)
 
 	// We don't need to process the response to a harvest command unless an
 	// error happened.  (Note that this may change if we have to support metric
 	// cache ids).
 	if nil == reply.Err {
+		addDataUsage(duc.duc, cmd.Name, len(cmd.Data), len(reply.Body))
 		return
 	}
+	// If we receive an error, the data was not stored into the collector
+	addDataUsage(duc.duc, cmd.Name, 0, len(reply.Body))
```

Comment:
Does the usage metric care about accepted transfers out to collector or just the amount of data being sent w/o regard for if it was accepted?

Reply:
After asking around, the answer I received is the former. It makes more sense for it to care about billable data, which is only that which is accepted.